### DISCUSSION TITLE
How to get results with where condition in Filament Admin Resource ?

### DISCUSSION THREAD

**Discussion by chahal26** [filamentphp/filament#3025] [Help] [answered]
I made a Resource for User model which works fine.

I have an `is_admin` field in the user's table which `returns 0 and 1`. I need users with `is_admin = 0` but for now, I am getting all of them.

Can I add a where condition in filament to get only the required fields?

**Comment by danharrin** [comment, marked as answer]
Override the `getEloquentQuery()` method and add the `where()` clause there:

https://filamentphp.com/docs/2.x/admin/resources/getting-started#customising-the-eloquent-query

**Reply by chahal26** [reply]
Thanks @danharrin
